src/i18n/en.json

```diff
@@ -964,49 +964,87 @@
-  "actionSidebar.tooltip.managePermissions.authority": "Whether they have the authority to take actions on their own, or require others with the same multi-sig permission to approve as well.",
-  "actionSidebar.tooltip.editColony.colonyName": "The display name of the Colony.",
-  "actionSidebar.tooltip.editColony.colonyLogo": "The logo of the Colony.",
-  "actionSidebar.tooltip.editColony.colonyDescription": "The short description of the Colony what it is about and it's purpose.",
+  "actionSidebar.tooltip.actionType": "Identifies the type of action.",
+  "actionSidebar.tooltip.decisionMethod": "Mechanism behind the decision-making process.",
+  "actionSidebar.tooltip.createdIn": "Team responsible for decision-making and action execution.",
+  "actionSidebar.tooltip.description": "Rationale and context behind the action.",
+  "actionSidebar.tooltip.singlePayment.from": "Team source of the funds.",
+  "actionSidebar.tooltip.singlePayment.recipient": "Member or address designated to receive the funds.",
+  "actionSidebar.tooltip.singlePayment.amount": "The amount of funds and token type for the payment.",
+  "actionSidebar.tooltip.singlePayment.additionalPayments": "Option to combine multiple payments with the same token.",
+  "actionSidebar.tooltip.advancedPayment.fundFrom": "Team source of the payment.",
+  "actionSidebar.tooltip.advancedPayment.recipient": "Member or address designated to receive the funds.",
+  "actionSidebar.tooltip.advancedPayment.amount": "Quantity and type of funds of the payment.",
+  "actionSidebar.tooltip.advancedPayment.claimDelay": "Time interval between the approved release of funds and being able to claim the funds, it works as a security delay or a way to schedule payments.",
+  "actionSidebar.tooltip.batchPayment.fundFrom": "Team source of the payment.",
+  "actionSidebar.tooltip.batchPayment.recipient": "Member or address designated to receive the funds.",
+  "actionSidebar.tooltip.batchPayment.amount": "Quantity and type of funds of the payment.",
+  "actionSidebar.tooltip.batchPayment.token": "Token type used for the payment.",
+  "actionSidebar.tooltip.splitPayment.fundFrom": "Team source of the payment.",
+  "actionSidebar.tooltip.splitPayment.description": "The way in which the payment should be divided.",
+  "actionSidebar.tooltip.splitPayment.amount": "Quantity and type of funds of the payment.",
+  "actionSidebar.tooltip.splitPayment.recipient": "Member or address designated to receive the funds.",
+  "actionSidebar.tooltip.splitPayment.percent": "Proportional division of funds for recipients.",
+  "actionSidebar.tooltip.stagedPayment.fundFrom": "Team source of the payment.",
+  "actionSidebar.tooltip.stagedPayment.recipient": "Member or address designated to receive the funds.",
+  "actionSidebar.tooltip.stagedPayment.milestone": "Specific phase or stage associated with the payment.",
+  "actionSidebar.tooltip.stagedPayment.amount": "The amount of funds and token type for the milestone.",
```

Comment:
Can this instead be: "Quantity and type of funds for the milestone."

Reply:
done